src/modules/getfoods.js

```diff
@@ -0,0 +1,72 @@
+import popUp from './popUp.js';
+import displayCounter from './counterFile.js';
+import { addLikeToItem, getLikesToItems } from './like.js';
+
+const displayData = async (arr) => {
+  const board = document.querySelector('.container-food-cards');
+  board.innerHTML = '';
+  arr.forEach((food) => {
+    const item = document.createElement('article');
+    item.id = food.idMeal;
+    item.classList.add('card-food');
+    item.innerHTML = `
+          <div class='card-title'>
+            <h5>${food.strMeal}</h5>
+          </div>
+          <button type='button' class='btn-recipe'>Recipe</button>
+          <div>
+          </div>
+          <div class='btn-liked'>
+            Like
+          </div>
+          <div class='card-img'>
+           <img src="${food.strMealThumb}" class='img-food'>        
+          </div>     
+      `;
+    /* eslint-disable */
```

Comment:
- [ ] Please remove the `eslint-disable` statement and try to fix it manually.

Reply:
Thanks for pointing that out @gemmen29, we'll work on it👏🏿

Reply:
This change has been done well...